timeline-chart/src/components/time-graph-annotation.ts

```diff
@@ -61,6 +61,8 @@ export class TimeGraphAnnotationComponent extends TimeGraphComponent {
             this.drawPlus(x, y, size);
         } else if (symbol == 'diamond') {
             this.drawDiamond(x, y, size);
+        } else {
```

Comment:
We have to make sure that when the symbol value is 'none' or undefined it draws nothing.

Then I'm not sure if it's better to have a default drawing only for known but unsupported symbols, or for any other symbol value.